src/commands/pull.js

```diff
@@ -0,0 +1,69 @@
+const chalk = require('chalk');
+const mkdirp = require('mkdirp');
+const fs = require('fs');
+const path = require('path');
+const tiged = require('tiged');
+const { ApifyCommand } = require('../lib/apify_command');
+const { getLoggedClientOrThrow } = require('../lib/utils');
+
+class PullCommand extends ApifyCommand {
+    async run() {
+        const { args } = this.parse(PullCommand);
+        const apifyClient = await getLoggedClientOrThrow();
+        const cwd = process.cwd();
+
+        const { actorId } = args;
+        const actor = await apifyClient.actor(actorId).get();
+        if (!actor) throw new Error(`Cannot find actor with ID '${actorId}' in your account.`);
+        const { name, versions } = actor;
+
+        // TODO: Check is sorted chronologically
+        const lastVersion = versions[versions.length - 1];
```

Comment:
It would be nice to be able to select the version with an argument. And maybe default to the version with the `latest` tag if there is one, instead of the last one in the array.

Reply:
<img width="464" alt="image" src="https://user-images.githubusercontent.com/20155254/234834025-f4836689-abb8-4891-a6e3-5987fe4b5dcb.png">
Well but there could be multiple 'latest' versions.

Reply:
Hm, true. Then it could sort those by semver.